tasks/02-uninstall.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: helm-uninstall
+spec:
+  steps:
+    - name: uninstall-instances
+      args:
+        - uninstall
+        - -n=default
+        - --ignore-not-found
+        - --wait
+        - kuadrant-instances
+      command:
+        - helm
+      env:
+        - name: KUBECONFIG
+          value: $(workspaces.shared-workspace.path)/kubeconfig
+      image: quay.io/rhn_support_azgabur/alpine/k8s:1.31.2
+      imagePullPolicy: IfNotPresent
+    - name: uninstall-operators
+      args:
+        - uninstall
+        - -n=default
+        - --ignore-not-found
+        - --wait
+        - kuadrant-operators
+      command:
+        - helm
+      env:
+        - name: KUBECONFIG
+          value: $(workspaces.shared-workspace.path)/kubeconfig
+      image: quay.io/rhn_support_azgabur/alpine/k8s:1.31.2
+      imagePullPolicy: IfNotPresent
+    - name: delete-cert-manager-namespace
+      args:
+        - delete
+        - --ignore-not-found
+        - ns
+        - cert-manager
+      command:
+        - kubectl
+      env:
+        - name: KUBECONFIG
+          value: $(workspaces.shared-workspace.path)/kubeconfig
+      image: quay.io/rhn_support_azgabur/alpine/k8s:1.31.2
+      imagePullPolicy: IfNotPresent
+    - name: delete-kuadrant-crd
```

Comment:
Gateway API gets deleted as part of Helm uninstall, Istio CRD does not. This can be problem if user changes istio providers on same cluster, I can add task to the pipeline that will delete Istio CRD. Good idea :+1:

Reply:
https://github.com/Kuadrant/testsuite-pipelines/pull/50/commits/53ffb50d4e03368ebe580b578e1ae41f65bbf85d

Reply:
Ok, no need to be part of this PR though